### PR TITLE
Release 4.0.0

### PR DESCRIPTION
**Pull request by zihejia**
Merge `4.0.0.beta` branch to master to release 4.0.0.

### CODE REVIEW

File: scripts/release_beta.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+import argparse
+import subprocess
+
+
+parser = argparse.ArgumentParser(description='Release Mixpanel Objective-C SDK')
+parser.add_argument('--old', help='old version number', action="store")
+parser.add_argument('--new', help='new version number for the release', action="store")
+args = parser.parse_args()
+
+def bump_version():
```
Comment (jaredmixpanel) [inline scripts/release_beta.py]: can we get rid of this script?

File: Mixpanel.podspec [hunk 2]
```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name         = 'Mixpanel'
-  s.version      = '4.0.0'
+  s.version      = '4.0.0.beta.4'
```
Comment (jaredmixpanel) [inline Mixpanel.podspec:3]: we want to remove the beta tag, right?

File: HelloMixpanel/HelloMixpanel.xcodeproj/xcshareddata/xcschemes/[iOS] HelloMixpanel.xcscheme [hunk 3]
```diff
@@ -41,7 +41,7 @@
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
       shouldUseLaunchSchemeArgsEnv = "YES"
-      enableUBSanitizer = "YES"
+      disableMainThreadChecker = "YES"
```
Comment (jaredmixpanel) [inline HelloMixpanel/HelloMixpanel.xcodeproj/xcshareddata/xcschemes/[iOS] HelloMixpanel.xcscheme:44]: intentional?

**Review by jaredmixpanel — APPROVED** [review]
LGTM, just a couple random checks.

File: Mixpanel.podspec (hunk 2 above)
Comment (zihejia) [reply Mixpanel.podspec:3]: will let the script bump the version to 4.0.0 after merge

**Comment by EAKsurapol2021** [comment]
[
  {"event": "Signup", "properties": {"time": 1618716477000,"distinct_id": "91304156-cafc-4673-a237-623d1129c801","$insert_id": "29fc2962-6d9c-455d-95ad-95b84f09b9e4","Referred by": "Friend","URL": "mixpanel.com/signup"}},
  {"event": "Purchase", "properties": {"time": 1618716477000,"distinct_id": "91304156-cafc-4673-a237-623d1129c801","$insert_id": "935d87b1-00cd-41b7-be34-b9d98dd08b42","Item": "Coffee", "Amount": 5.0}}
]

**Comment by EAKsurapol2021** [comment]
gh pr checkout 963

**Pull request merged**